jest.config.js

```diff
@@ -9,16 +9,8 @@ module.exports = {
     '<rootDir>/packages/test-renderer/dist',
     '<rootDir>/test-utils',
   ],
-  // coverageThreshold: {
-  //   global: {
-  //     statements: 80,
-  //     branches: 68,
-  //     functions: 80,
-  //     lines: 80,
-  //   },
-  // },
   coverageDirectory: './coverage/',
-  collectCoverage: true,
+  collectCoverage: false,
```

Comment:
Turned this to false since it was on for all unit tests, annoying! It's still enabled for `yarn test`.